workers/unity/Packages/com.improbable.gdk.core/Systems/SpatialOSReceiveSystem.cs

```diff
@@ -30,7 +30,7 @@ protected override void OnUpdate()
             try
             {
                 Profiler.BeginSample("GetMessages");
-                worker.GetMessages();
+                worker.Advance();
```

Comment:
I disagree with this function name. It gives no indication of what it does.

Reply:
Changed to `Tick()`